guide/src/reference/types/jsvalue.md

```diff
@@ -4,6 +4,25 @@
 |:---:|:---:|:---:|:---:|:---:|:---:|:---:|
 | Yes | Yes | No | Yes | No | No | Any JavaScript value |
 
+## `&[JsValue]` slices
+
+You can pass `&[JsValue]` slices from Rust to JavaScript. The slice is passed as
+a JavaScript `Array`. Since this is a borrowed slice, JavaScript receives a
+temporary view of the values - the array is only valid for the duration of the
+function call.
```

Comment:
So, apologies for the confusion.
I was under the impression that you are returning a view into the Wasm memory here, which is what `Clamped` is for.
(now that I'm thinking about it this wouldn't even work properly, because the `externref`s aren't in the linear memory)

However, looking at the code you clearly already do what I was suggesting: creating a JS `Array` and cloning every `externref`.

Why would JS receive only a temporary view here, this implies that there is some lifetime associated which it is not.


```suggestion
You can pass `&[JsValue]` slices from Rust to JavaScript. The slice is passed as
a JavaScript `Array`.
```

Am I missing something here?